source/whats-new.txt

```diff
@@ -55,6 +56,49 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.4:
+
+What's New in 6.4
+-----------------
+
+The {+driver-short+} v6.4 release includes the following features:
+
+- When multiples ``mongos`` instances are available, different servers are used
+  for read and write retry attempts.
+
+- Caches AWS credentials at the client level, rather than for each
+  authentication.
+
+- Upgrades to using BSON 6.4.0. For details about the new BSON features, see the
+  release notes for `BSON 6.3.0
+  <https://github.com/mongodb/js-bson/releases/tag/v6.3.0>`__ and `BSON 6.4.0
+  <https://github.com/mongodb/js-bson/releases/tag/v6.4.0>`__.
+  
+- Read operations that result in an ``ExceededTimeLimit`` error are retried.
+
+- Fixes a request issue related to :ref:`TLS sockets <node-connect-tls>` and
+  :manual:`KMS Providers
+  </core/queryable-encryption/fundamentals/kms-providers/>`.
+
+- Fixes the base64 padding on the ``saslContinue`` command to allow for mongosh
+  authentication.
+
+- Types ``countDocuments`` using ``Filter<Schema>`` rather than ``Document``,
+  which enables autocompletion and helps prevent downstream typing issues.
+
+- Fixes a type error in the ``$addToSet`` option of the ``bulkWrite`` command.
+  The ``$addToSet`` validation is skipped when users extend their types from
+  ``Document`` or ``any``, or use properties of any type.
```

Comment:
Passive voice. The style guide mentions using 2nd person rather than writing "users ...": https://www.mongodb.com/docs/meta/style-guide/writing/write-to-the-user/

```suggestion
- Fixes a type error in the ``$addToSet`` option of the ``bulkWrite`` command.
  The command skips ``$addToSet`` validation when you extend your types from
  ``Document`` or ``any``, or use properties of any type.
```